python/activity_stream/widget_note.py

```diff
@@ -49,23 +51,87 @@ def __init__(self, parent):
         self._general_widgets = []
         self._reply_widgets = []
         self._attachment_group_widgets = {}
-                
+        self._selected = False
+        self._attachments = []
+        self._show_note_links = True
+
+        self.setStyleSheet("#frame { border: 1px solid transparent }")
```

Comment:
is this so that it doesn't "jump" when it gets highlighted? May be worth adding a comment explaining.